machine-types/grid/base.pan

```diff
@@ -1,277 +1,140 @@
-############################################################
-#
-# template machine-types/grid/base
-#
 # Define base configuration of any type of gLite node.
 # Can be included several times.
 #
 # This template is included one. As it does VO initialization
 # the VO list and parameters cannot be redefined if included twice.
-#
-# RESPONSIBLE: Michel Jouvin
-#
-############################################################
 
 unique template machine-types/grid/base;
 
 variable CONFIGURE_VOS ?= false;
 
-
-# Include static information and derived global variables.
-variable SITE_DB_TEMPLATE ?= if_exists('pro_site_databases');
-variable SITE_DB_TEMPLATE ?= 'site/databases';
-include { SITE_DB_TEMPLATE };
-variable SITE_GLOBAL_VARIABLES ?= if_exists('site/global_variables');
-variable SITE_GLOBAL_VARIABLES ?= if_exists('pro_site_global_variables');
-variable SITE_GLOBAL_VARIABLES ?= error('site/global_variables not found');
-include { SITE_GLOBAL_VARIABLES };      # Default value for net params
-
 # When true, only the initial part of NFS configuration is done during base OS configuration.
 variable OS_POSTPONE_NFS_CONFIG ?= false;
 
-
-#
-# define site functions
-#
-variable SITE_FUNCTIONS_TEMPLATE ?= if_exists('pro_site_functions');
-variable SITE_FUNCTIONS_TEMPLATE ?= 'site/functions';
-include { SITE_FUNCTIONS_TEMPLATE };
-
-#
-# profile_base for profile structure
-#
-include { 'quattor/profile_base' };
-
-#
-# NCM core components
-#
-include { 'components/spma/config' };
```

Comment:
This include seems to be replaced by `'components/spma/functions';` in the [core-init.pan](https://github.com/quattor/template-library-standard/blob/master/machine-types/core-init.pan) file. Is that right?

Reply:
@jouvin?

Reply:
Sorry for the lack of follow-up. I'll try do to this today.

Reply:
Thanks, the rest of this looks good, just need an answer to this question.

Reply:
I checked but this line was removed a while ago. For some reason, it seems that GH didn't realize it...